.editorconfig

```diff
@@ -3,7 +3,7 @@ root = true
 
 [*]
 indent_style = space
-indent_size = 4
+indent_size = 2
```

Comment:
Is it intentional to change the indent from 2 to 4? All our source files are still using 4 spaces, and I think this is consistent with other projects like ZE.

Reply:
I saw some warnings on the readme when performing 4 indent vs 2 indent for some reason, I'm not sure if there is some issue with how the linter is viewing this vs how many tabs are made which led me to change this...

Reply:
I switched it back to 4, perhaps we can see later why that warning for indents shows with 4 on md files